whisk/client.go

```diff
@@ -51,6 +51,12 @@ const (
     DEFAULT_HTTP_TIMEOUT = 30
 )
 
+type ClientInterface interface {
```

Comment:
there are actually more methods for Client than are defined in this interface.  i'm thinking that if this will be the interface for all future method mocks, then include all methods here.. ?

Reply:
These are the bare minimum needed to get the client to compile.